README.md

```diff
@@ -57,12 +57,22 @@ Those two urls are optional:
 
 Grafana issues this request on 
 
-1. _Variables > New/Edit_ page. `Query` field value is passed in a body as 
+1. _Variables > New/Edit_ page. `Query` field value is passed in a body as below (template variables are expanded as regex by default)
 
 ```json
 { "target": "query field value" }
 ```
 
+Alternatively, if you provide a full valid JSON string in the query field, it will be passed in the body as a native JSON object, and template variables will be expanded as a JSON array. For example, selecting two items in a multi-field dropdown `$myservers`
+```string
+{"selectedservers":$myservers}
```

Comment:
can we add a test? What happens when no value/one entry is selected?

Reply:
It uses the default behavior of the Grafana interpolation for JSON. When one is selected, it provides it as a string:
![image](https://user-images.githubusercontent.com/529833/111911693-f3ed0600-8a3c-11eb-994e-16daf62e39b6.png)

And when nothing is selected, it leaves the string empty:
![image](https://user-images.githubusercontent.com/529833/111911660-dae45500-8a3c-11eb-90ac-f1ad08b98c70.png)

I also want to add a test, I'm just having trouble figuring out how that works for the input of this field. I will take another look.